### PR TITLE
Update math.py

### LINKED ISSUE #10117: exp

### PR DESCRIPTION
**Pull request by Sravanthgithub**
close #10117

### CODE REVIEW

**Comment by Sravanthgithub** [comment]
@karalleyna any more changes to be made? or is this looking good?

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

File: ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py [hunk 1]
```diff
@@ -93,6 +93,31 @@ def test_tensorflow_tan(
         on_device=on_device,
         x=x[0],
     )
+    
+    
+# exp
+ @handle_frontend_test(
```
Comment (hirwa-nshuti) [inline ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py:99]:
```suggestion
@handle_frontend_test(
```

**Review by hirwa-nshuti — COMMENTED** [review]
Hey @Sravanthgithub 

Sorry for the late review somehow this slipped under our radar.
All looks good to me and I'm going to commit this minor suggestion and merge.

Happy coding😊

**Pull request merged**